Download Allow List Process

hhund opened the pull request:
Adds process to download and update the allow list from a known server via a full Bundle url.
Renames remaining occurrences of "whitelist" into "allow list"

closes #118

wetret reviewed (approved):
Currently you need to know the bundle URL to be able to download the allow list. It would also be nice if you could only specify the TTP where the BPE should look for the bundle. After that, a search for the latest version will be executed and if there is a bundle, it will be downloaded.

hhund commented:
I implemented it intentionally a little bit inconvenient. I think we should add support for BPMN user tasks to enable reviewing a downloaded transaction bundle before executing it. I added #139 to track supporting user tasks and #140, #141 and #142 to modify the existing resource "download" processes.
